qa/rolling-upgrade/src/test/java/org/opensearch/upgrades/IndexingIT.java

```diff
@@ -267,7 +267,7 @@ public void testIndexingWithSegRep() throws Exception {
                     .put(IndexMetadata.SETTING_REPLICATION_TYPE, ReplicationType.SEGMENT)
                     .put(
                         EngineConfig.INDEX_CODEC_SETTING.getKey(),
-                        randomFrom(CodecService.DEFAULT_CODEC, CodecService.BEST_COMPRESSION_CODEC, CodecService.LUCENE_DEFAULT_CODEC)
```

Comment:
Outside the scope of current PR, we should add ZSTD and ZSTD_NO_DICT codes as well here.

Reply:
PR to add rest of codecs: https://github.com/opensearch-project/OpenSearch/pull/8907